### DISCUSSION TITLE
what is the role of  "adaptive_normalization" module?

### DISCUSSION THREAD

**Discussion by dhtheuno** [snakers4/silero-vad#490] [Q&A] [answered]
I am currently looking through the .jit structure of the model, and it seems like the adaptive_normalization module is present but never used in the forward function of any other modules. Do you use it during training?

**Comment by adamnsandle** [comment, marked as answer]
`adaptive_normalization` is a stale module from previous models. v5 model contains it but never uses. We'll get rid of it in future releases.

**Reply by dhtheuno** [reply]
Thank you for the fast response! I noticed that you are using the MobileOne block, which is a reparameterized model. Are you planning to release the entire training model along with the training procedure?

**Comment by dayfinihzy** [comment]
How can I see the structure of the model by looking through the .jit file? I just use 'load_jit' and 'print', but I can not find "adaptive_normalization" in the result.  Looking forward to receiving your reply. Thank you!
